### PR TITLE
cleaning up duplicate steve entries and blank space

### PR DESCRIPTION
**Pull request by evilearthwormjim**
Please have a look

### CODE REVIEW

**Comment by reufis** [comment]
looks good

**Review by Kulkav9 — COMMENTED** [review]
Test Review

File: ReadMe.txt [hunk 1]
```diff
@@ -2,17 +2,5 @@ Reuben's branch
 Julie's sentence.
 Steve's Change
 This is the second sentence added by Peter
-Steve's Change
-
-Steve's Change
-
```
Comment (JulieKoz) [inline ReadMe.txt:8]: Looks good to me

Comment (gottig) [inline ReadMe.txt:8]: I need these empty lines.

File: ReadMe.txt [hunk 2]
```diff
@@ -2,17 +2,5 @@ Reuben's branch
...
-
-
-
-
-
-
-
-
```
Comment (reufis) [inline ReadMe.txt:16]: I always delete Steve's code

**Pull request merged**